### PR TITLE
Update data-classification-overview.md

### PR DESCRIPTION
**Pull request by sudeepghatak**

### CODE REVIEW

File: microsoft-365/compliance/data-classification-overview.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ description: "The data classification dashboard gives you visibility into how mu
...
+As a Microsoft 365 administrator or compliance administrator, you can evaluate and then tag content in your organization in order to control where it goes, protect it no matter where it is and to ensure that it is preserved and deleted according to your organization's needs. You do this through the application of [sensitivity labels](sensitivity-labels.md), [retention labels](retention.md#retention-labels), and sensitive information type classification. There are various ways to do the discovery, evaluation and tagging, but the end result is that you may have very large number of documents and emails that are tagged and classified with one or both of these labels. After you apply your retention labels and sensitivity labels, you'll want to see how the labels are being used across your tenant and what is being done with those items. The data classification page provides visibility into that body of content, specifically:
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/data-classification-overview.md]:
```suggestion
As a Microsoft 365 administrator or compliance administrator, you can evaluate and then tag content in your organization in order to control where it goes, protect it no matter where it is, and ensure that it is preserved and deleted according to your organization's needs. You do this through the application of [sensitivity labels](sensitivity-labels.md), [retention labels](retention.md#retention-labels), and sensitive information type classification. There are various ways to do the discovery, evaluation, and tagging, but the end result is that you may have a very large number of documents and emails that are tagged and classified with one or both of these labels. After you apply your retention labels and sensitivity labels, you'll want to see how the labels are being used across your tenant and what is being done with those items. The data classification page provides visibility into that body of content, specifically:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @sudeepghatak, kindly also commit the additional suggested changes. Many thanks.

**Comment by yogkumgit** [comment]
@sudeepghatak Gentle reminder.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by yogkumgit** [comment]
@chrfox Please merge this PR. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**